src/core/storage/sqliteStorage/migrations/updates/version-0.0.1.ts

```diff
@@ -0,0 +1,91 @@
+import { SQLiteDBConnection } from "@capacitor-community/sqlite";
+import { SqliteStorage } from "../../sqliteStorage";
+import { resolveTagsFromDb } from "../../utils";
+import { StorageRecord } from "../../../storage.types";
+import { CredentialMetadataRecord } from "../../../../agent/records";
+import { deserializeRecord } from "../../../utils";
+
+export const DATA_V001 = {
+  version: "0.0.1",
```

Comment:
Isn't the original `0.0.1`? https://github.com/cardano-foundation/cf-identity-wallet/blob/2fc8da0ca83332053c682984465fac2335fb1951/src/core/storage/sqliteStorage/migrations/version-0.0.1.ts

Reply:
No, this is version 0.0.1 for migration ts (update prefix credential)